portal/static/js/admin.js

```diff
@@ -45,7 +45,9 @@
             configured: false,
             initIntervalId: 0,
             sortFilterEnabled: false,
+            isAdmin: false,
             userId: null,
+            userRoles: [],
```

Comment:
Do you still need to save the roles here if they're already cached in localStorage?

Reply:
@ivan-c Ivan, yea, it will be, it is just that having a property here allows other methods in the Admin Vue instance to have access to it.